### PR TITLE
enable CRIU_Image_Upload on ub22 amd

### PR DESCRIPTION
**Pull request by llxia**

### CODE REVIEW

**Comment by llxia** [comment]
[CRIU_Image_Upload](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/CRIU_Image_Upload/279/)

**Comment by llxia** [comment]
@AdamBrousseau could you review this change? Thanks

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -560,10 +560,11 @@ class Build {
...
                     'x86-64_linux' : [
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.ubuntu.22&&hw.arch.x86.broadwell'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.broadwell'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.amd'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.skylake']
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.ubuntu.22&&hw.arch.x86.broadwell'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.ubuntu.22&&hw.arch.x86.amd'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.broadwell'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.amd'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.skylake']
```
Comment (AdamBrousseau) [inline pipelines/build/common/openjdk_build_pipeline.groovy:567]: Should we refactor the repeated labels into a variable?

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -403,7 +403,7 @@ class Build {
                             if (additionalTestLabel != '') {
                                 additionalTestLabel += '&&'
                             }
-                            additionalTestLabel += 'sw.tool.podman&&(sw.os.ubuntu.22||sw.os.rhel.8)'
```
[the remark this answers is not in the document]
Comment (llxia) [reply pipelines/build/common/openjdk_build_pipeline.groovy:406]: Yes, we have issues with other OS. (i.e., ub20 and rhel9)

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (llxia) [reply pipelines/build/common/openjdk_build_pipeline.groovy:567]: Yes, we can, but this code will be removed eventually. The plan is to move the logic to the test pipeline. See https://github.com/adoptium/aqa-tests/issues/4465

**Pull request merged**